curriculum/js1/3.js

```diff
@@ -5,8 +5,11 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (x) => {
+  if(x<=0){
+    return ""
+  }
+  return solution(x-1)+"hello";
```

Comment:
this is really inefficient. You don't want to do this when using recursion. Hint:
```
const solution = (x, result="") => {
  if(x<=0) return result;
  ....
```